projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -153,6 +153,14 @@
     }
   }
 
+  .datagrid-overflow-ellipsis .datagrid-row .datagrid-scrolling-cells {
+    > .datagrid-cell:not(.datagrid-row-actions, .datagrid-select, .datagrid-expandable-caret) {
```

```suggestion
    .datagrid-overflow-ellipsis .datagrid-row .datagrid-scrolling-cells .datagrid-cell {
```
These classes (.datagrid-scrolling-cells and .datagrid-row-sticky) are mutually exclusive.  Since selector is wrapped with .datagrid-scrolling-cells, there is no need to check for :not pseudo class as that combination will never exist with `.datagrid-scrolling-cells .datagrid-cell`.

---

Good catch corrected in commit 0a527ad.